extensions.json

```
@@ -269,6 +274,11 @@
       "repository": "https://github.com/felixfbecker/vscode-php-pack",
       "version": "1.0.2"
     },
+    {
+      "id": "foam.foam-vscode",
+      "repository": "https://github.com/foambubble/foam-vscode",
+      "version": "0.1.4"
```

The current version seems to be `0.1.5`. Also, it seems safer to publish it from the corresponding Git tag (`v.0.1.5`) as opposed to from the repo's default Git branch:

```suggestion
      "version": "0.1.5",
      "checkout": "v0.1.5"
```